combinations/hash.tsv

```diff
@@ -139,3 +139,4 @@ biopython=1.78,openpyxl=3.0.7,pyfastx=0.8.2,numpy=1.20.2,pandas=1.2.4
 python=3.9,biopython=1.78,openpyxl=3.0.7,pandas=1.2.4,rich=10.2.2,typer=0.3.2,xlsxwriter=1.4.3
 asciigenome=1.16.0,bedtools=2.30.0
 r-recetox-aplcms=0.9.3,r-base=4.1.0,r-arrow=4.0.1	quay.io/bioconda/base-glibc-debian-bash:latest	0
+python=3.9.1,pandas=1.3.0,seaborn=0.11.0
```

Comment:
can we use python 3.9? maybe that is then more general in the long run?

Reply:
ok